components/tabs/src/atoms/TabList/index.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
```

Comment:
🚲 please follow naming convention for files and folders used elsewhere , i.e. `TabList` should be `tab-list`

Reply:
Related issue raised #587